servers/express.js

```diff
@@ -34,20 +57,49 @@ app.all('/*', function (req, res, next) {
 
 app.use(express.static(path.join(__dirname, '..', 'front-end', 'build')));
 
+app.use(bodyParser.json());
+
+app.post('/api/upload', upload.any(), (req, res) => {
```

Comment:
is there any other way to get the files that are available? 

maybe `app.post('/api/logfiles')` uploads files while `app.get(`/api/logfiles')` lists them?